composer.json

```diff
@@ -11,17 +11,21 @@
     ],
     "require": {
         "php": ">=5.5",
+        "symfony/yaml": "~2.5",
```

Comment:
Not needed due to `behat/behat`, after next release we need to fix the BC break.